class.jetpack-options.php

```diff
@@ -132,7 +156,13 @@ public static function is_valid( $name, $group = null ) {
 	 */
 	public static function get_option( $name, $default = false ) {
 		if ( self::is_valid( $name, 'non_compact' ) ) {
-			return get_option( "jetpack_$name", $default );
+			if ( self::is_network_option( $name ) ) {
+				return get_site_option( "jetpack_$name", $default );
+			}
+			else {
```

Comment:
Also, as the prior conditional has a return statement, this really doesn't need to be in an `else` at all.  It could just have the prior if as a short circuit.